CHANGES.md

```diff
@@ -1,3 +1,5 @@
+- [IMPROVED] Removed HTTP GET request when instantiating `Database` objects
```

Comment:
I think we should word this more along the lines of 

`[IMPROVED] Improved performance by reducing the number of requests made when instantiating Database objects.`